lighthouse-core/test/gather/gatherers/service-worker-test.js

```diff
@@ -40,7 +40,7 @@ describe('Service Worker gatherer', () => {
           return Promise.resolve({versions});
         }
       },
-      url
+      url: {finalUrl: url}
```

Comment:
I believe `url` was correct here (this is the `options` object, not `artifacts`). No error was caused by changing this because the gatherer doesn't actually use `options.url`